src/app/counter-group/counter-group.component.html

```diff
@@ -0,0 +1,19 @@
+<div>
+    <counter 
```

Comment:
good code style of writing such as define remove and reset event to receive counter id